app/src/main/java/com/willowtree/vocable/room/VocableDatabase.kt

```diff
@@ -4,17 +4,26 @@ import android.content.Context
 import androidx.room.Database
 import androidx.room.Room
 import androidx.room.RoomDatabase
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
 
-@Database(entities = [Category::class, Phrase::class], version = 1)
+@Database(entities = [Category::class, Phrase::class], version = 2)
 abstract class VocableDatabase : RoomDatabase() {
 
     companion object {
         private var vocableDatabase: VocableDatabase? = null
 
+        private val MIGRATION_1_2: Migration = object : Migration(1, 2) {
+            override fun migrate(database: SupportSQLiteDatabase) {
+                database.execSQL("DROP INDEX index_Phrase_utterance")
```

Comment:
We should put a comment on this migration to specify what it's doing/why it's needed

Reply:
`// Need to drop this index from the Phrase table to migrate`

sound okay?

Reply:
I would go with something like -
`// Phrases no longer have to have unique utterances`